src/local.lambda.ts

```diff
@@ -35,10 +37,13 @@ export class LocalLambda {
     this.binaryContentTypesOverride = new Set(config.binaryContentTypesOverride ?? defaultBinaryContentTypeHeaders);
     this.pathParamsPattern = config.pathParamsPattern ?? DefaultPathParamsPattern;
     this.app = express();
+    this.requestContext = config.requestContext ?? {};
   }
 
   run(): void {
     this.app.all(`${this.pathParamsPattern}*`,async (request: Request, response: Response) => {
+      // create a copy of requestContext to avoid accidental mutation
+      const copyOfRequestContext = _.cloneDeep(this.requestContext);
```

Comment:
looks like `deep clone` is the only thing we use from `lodash`. Given the `reuqestContext` is a regular data object and not created by javascript constructs, I guess it is safe to use `const copyOfRequestContext = JSON.parse(JSON.stringify(this.requestContext))` here? 

Please let me know if it makes sense.

Reply:
Hey @vcfvct , I've addressed this.